### PR TITLE
Enable Prometheus in every microservice

### PR DESCRIPTION
**Pull request by antoinebhs**
First, this is to access Tomcat metrics in every micro-service. 

Move common Spring-Boot config in powsybl/powsybl-ws-commons#49

### CODE REVIEW

File: k8s/resources/common/case-server-service.yaml
Comment (Tristan-WorkGH) [inline k8s/resources/common/case-server-service.yaml:1]: Wouldn't it be better, like how we do with `deployment` object, to start a common `k8s/components/gridsuite/patches/deployments-springboot-service.yaml` to store common config like labels?

File: k8s/resources/common/config/loadflow-server-application.yml [hunk 1]
```diff
@@ -1,8 +1,2 @@
 loadflow:
   default-provider: OpenLoadFlow
-
-management:
-  endpoints:
-    web:
-      exposure:
-        include: prometheus, health, info
```
Comment (Tristan-WorkGH) [inline k8s/resources/common/config/loadflow-server-application.yml:8]: Why did you delete it?
It's needed to control which parts we initialize & expose, and the docker part use also this file.

Comment (Tristan-WorkGH) [inline k8s/resources/common/config/loadflow-server-application.yml:8]: You can move it to [`k8s/resources/common/config/common-application.yml`](https://github.com/gridsuite/deployment/blob/enable-prometheus-all-ms/k8s/resources/common/config/common-application.yml) to de-duplicate configuration.

**Review by Tristan-WorkGH — CHANGES_REQUESTED** [review]

File: k8s/resources/common/case-server-service.yaml
Comment (Tristan-WorkGH) [reply k8s/resources/common/case-server-service.yaml:1]: or in [`k8s/components/gridsuite/patches/services-springboot.yaml`](https://github.com/gridsuite/deployment/blob/enable-prometheus-all-ms/k8s/components/gridsuite/patches/services-springboot.yaml)

File: k8s/resources/common/config/loadflow-server-application.yml (hunk 1 above)
Comment (antoinebhs) [reply k8s/resources/common/config/loadflow-server-application.yml:8]: I actually moved it even higher in the hierarchy, see https://github.com/powsybl/powsybl-ws-commons/pull/49

File: k8s/resources/common/case-server-service.yaml
Comment (antoinebhs) [reply k8s/resources/common/case-server-service.yaml:1]: Yes, agreed!

File: k8s/resources/common/config/loadflow-server-application.yml (hunk 1 above)
Comment (Tristan-WorkGH) [reply k8s/resources/common/config/loadflow-server-application.yml:8]: oh okay, it would be good to wait that PR49 is merged+released and all services upgraded to latest powsybl-ws-common version to not loose current config, as deployment repository is automatically updated to main head.

I added a reference between the PR to keep information.

**Review by Tristan-WorkGH — APPROVED** [review]
OK but need to wait release of powsybl-ws-common released and used in services before merge.

**Review by SlimaneAmar — APPROVED** [review]
